createdisk-library.sh

```diff
@@ -116,7 +116,7 @@ function prepare_hyperV() {
     # Install the hyperV rpms to VM
     ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora.repo'
     ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora-updates.repo'
-    ${SSH} core@${vm_ip} -- 'sudo rpm-ostree install --allow-inactive hyperv-daemons'
+    ${SSH} core@${vm_ip} -- 'sudo rpm-ostree install --allow-inactive hyperv-daemons cockpit-bridge cockpit-ws cockpit-podman'
```

Comment:
Why is this done in prepare_hyperV? Is this specific to the Windows bundle? The commit log does not say anything about it. What is the impact on bundle size?

Reply:
> Why is this done in prepare_hyperV? Is this specific to the Windows bundle?

No it is not only specific to windows bundle but as of now this is the place where we enable the repo and install the packages so we might need to separate this out.

> The commit log does not say anything about it. What is the impact on bundle size?

@cfergeau  It increase from 641 => 671MB, will mention in the commit message.